src/meta/src/barrier/recovery.rs

```diff
@@ -352,7 +347,7 @@ where
                     }),
                 })
                 .await
-                .map_err(RwError::from)
+                .map_err(MetaError::from)
```

Comment:
No need to map here?

Reply:
We need it here. Because the closure has another `?` operator which returns `RpcError`, we should use map here to let compiler know the return error type of the closure is `MetaError` but not `tonic::Status`.